README.md

```diff
@@ -86,20 +84,124 @@ code because of all the basic column-renamings and the full listing of health co
+are now running our refreshes in *dev_mode* using the dataflow_beam runner.
+The Dagster project we added for DAP includes 3 different modes of operation:
+1. `local_mode` uses local_beam_runner
+1. `dev_mode` uses dataflow_beam_runner
+1. `prod_mode` uses dataflow_beam_runner
+
+For now, we can call individual pipelines manually with any runner and then call the tsv script to write files locally.
+The general structure for the SBT call to kick off one of our pipelines:
+* `sbt "{scala_project} {target_class} {sbt parameters} {dataflow parameters}"`
+* Here is sample an sbt for the Environment extraction pipeline:
+    >sbt "dog-aging-hles-extraction/runMain org.broadinstitute.monster.dap.environment.EnvironmentExtractionPipeline 
+    --apiToken=foo --pullDataDictionaries=false --outputPrefix=bar --runner=dataflow --project=baz --region=us-central1
+    --workerMachineType=n1-standard-1 --autoscalingAlgorithm=THROUGHPUT_BASED --numWorkers=4 --maxNumWorkers=8 
+    --experiments=shuffle_mode=service --endTime=qux"
+
+Alternatively we can also execute a Dagster pipeline which will run all the pipelines and write local TSVs and will 
+use a runner based on the mode.
+
+### Manual Refresh (Dagster)
+1. Navigate to the *orhcestration* subdirectory
+2. Poetry installation
+    * `poetry install` to install your project’s package
+    * `poetry update` to get the latest versions of the dependencies and to update the poetry.lock file
+2. Update config entries in orchestration/dev_refresh.yaml:
+    * Update the `refresh_directory` resource config entry (_this should be a GCS path_)
+    * Update the `working_dir` config entry for the *write_outfiles* solid
+    * Update the `end_time`, `pull_data_dictionaries`, and `api_token` config entries for each extract solid
+3. Execute the pipeline:
+    * `poetry shell` spawns a shell within the project's virtual environment
+    * `dagster pipeline execute -f repositories.py -a repositories -p refresh_data_all --mode dev -c dev_refresh.yaml`
+4. Files should appear within a "tsv_output" relative to the `working_dir` specified in the config.
 
-Manually processing data on our local machines obviously isn't ideal, but it should scale well enough
-as long as HLES stays on the order of a few thousand participants. Once we cross into the area of 10K+
-participants, we'll need to either push the transformation logic into the cloud, or (ideally) stop doing
-things manually altogether in favor of the official TDR<->Workspace integration.
+### Data Delivery to DAP
+When the hles_data volume was much smaller, we were able to load the final TSVs directly to Terra using the UI prompts.
+Once the data grew too big to load via browser UI, we started utilizing the field engineering script to upload large entities. 
+While still building out the TDR - workspace integration for DAP, we are using a Google bucket to deliver refreshed TSVs.
```

Comment:
⛏️ I think we're using the "<->" notation for the integration above, so for consistency I suggest:
```suggestion
While still building out the TDR <-> workspace integration for DAP, we are using a Google bucket to deliver refreshed TSVs.
```